### ISSUE TITLE
"Red Frame" not visible on production connection when editing stored procedure

### ISSUE THREAD

**Issue by Grisgram** [dbeaver/dbeaver#22693]
### Description

Hey there,
I really love the coloring of a table when on production - the dark red holds up higher attention to what you're doing.

I just found out, when I edit a stored procedure on production (which one should only do if you have a _really_ good reason!), then there is no red frame drawn around the sp-editor. just the connection in the top toolbar shows red, but the editor itself has no coloring.

### DBeaver Version

latest community

### Operating System

win10

### Database and driver

mysql/mariadb

### Steps to reproduce

open a connection that is set to "production" from the connection type, so you have red coloring
open a table - you see a red border around the table, warning you, that's production
now double-click a stored procedure on that connection -> no red frame covering the editor

thanks for reading!

**Comment by LonwoLonwo** [comment]
Hello @Grisgram 

Could you please provide a screen?
Unfortunately, I can't reproduce it for now.

**Comment by Grisgram** [comment]
Sure, see here two pictures, one from a table view and one from editing a stored procedure.
![image](https://github.com/dbeaver/dbeaver/assets/19487451/6219a904-bd43-4d39-ae42-d3a9e50d0344)
![image](https://github.com/dbeaver/dbeaver/assets/19487451/ccbebd7b-ad7e-44a1-a5ea-ad75454ab914)

**Comment by LonwoLonwo** [comment]
And this procedure is definitely behaving to your production connection?

If so, could you please provide the log files? You can find instructions here: https://dbeaver.com/docs/wiki/Log-files/. Thank you.

**Comment by Grisgram** [comment]
yes it is, here a pic with the active (red) connection in the main toolbar
![image](https://github.com/dbeaver/dbeaver/assets/19487451/7b2db33c-69ef-4913-9e97-16472be9db7b)

in the log file there is nothing useful (debug-log i mean),

```
!ENTRY org.jkiss.dbeaver.model 2 0 2024-02-04 20:02:03.884
!MESSAGE Some data source listeners are still registered: [node://api/datasources]
!SESSION 2024-02-05 17:58:20.684 -----------------------------------------------
eclipse.buildId=unknown
java.version=17.0.6
java.vendor=Eclipse Adoptium
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en
Framework arguments:  -eclipse.keyring C:\Users\mikeb\AppData\Roaming\DBeaverData\secure\secure_storage
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.ui 2 2 2024-02-05 17:58:24.643
!MESSAGE Invalid preference category path: org.eclipse.debug.ui.ConsolePreferencePage (bundle: org.eclipse.ui.console, page: org.eclipse.ui.internal.console.ansi.preferences.AnsiConsolePreferencePage)

!ENTRY org.jkiss.dbeaver.model 2 0 2024-02-05 17:58:41.964
!MESSAGE Some data source listeners are still registered: [node://api/datasources]
!SESSION 2024-02-05 17:59:19.826 -----------------------------------------------
eclipse.buildId=unknown
java.version=17.0.6
java.vendor=Eclipse Adoptium
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en
Framework arguments:  -eclipse.keyring C:\Users\******\AppData\Roaming\DBeaverData\secure\secure_storage
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.ui 2 2 2024-02-05 17:59:26.098
!MESSAGE Invalid preference category path: org.eclipse.debug.ui.ConsolePreferencePage (bundle: org.eclipse.ui.console, page: org.eclipse.ui.internal.console.ansi.preferences.AnsiConsolePreferencePage)

```

**Comment by LonwoLonwo** [comment]
Oh! I understood. This issue exists only for the dark theme, not the Light one.

And I also see the absence of production colors for tables and views editors.

**Comment by Grisgram** [comment]
ok -- so the information is enough to fix it, do I understand you correctly?
would be great! 
thx, Gris

**Comment by LonwoLonwo** [comment]
Yes, thanks.

**Comment by E1izabeth** [comment]
Can't reproduce anymore

<img width="1088" height="709" alt="Image" src="https://github.com/user-attachments/assets/7b4d42f9-23ca-4849-b2db-257cdc955a4f" />
